### ISSUE TITLE
Understanding SC 1.4.11 - Incorrect alt text

### ISSUE THREAD

**Issue by honoka-h** [w3c/wcag#804]
The image below is showing 4 radio buttons, but in the alt text, it says there are 3 radio buttons. It seems like the alt text should be adjusted.

https://www.w3.org/WAI/WCAG21/Understanding/img/radio-custom.png

```html
<img alt="Three radio buttons, the first a plain circle marked unselected. The second shows the circle filled with the same color as the border. The third has a slightly darker filling than the border." src="img/radio-custom.png">
```

**Comment by alastc** [comment]
Thanks for that, I pushed [an update](https://github.com/w3c/wcag/commit/d6ba73b05e1590194d2e4aefb38953e8ad827ebf) straight to master.
